data/moves.js

```diff
@@ -7074,10 +7076,12 @@ let BattleMovedex = {
 		priority: 0,
 		flags: {},
 		isMax: "Alcremie",
-		onAfterHit(target, source) {
-			for (let pokemon of source.side.active) {
-				pokemon.heal(pokemon.maxhp);
-			}
+		self: {
+			onAfterHit(target, source) {
+				for (let pokemon of source.side.active) {
+					pokemon.heal(pokemon.maxhp);
```

Comment:
While you're here, this should be `this.heal(pokemon.maxhp, pokemon, pokemon);`. See https://www.smogon.com/forums/posts/8287090/

Reply:
thanks!

Reply:
thanks!